devtools/cli/src/test/java/io/quarkus/cli/CliProjectGradleTest.java

```diff
@@ -106,6 +107,8 @@ public void testCreateAppDefaults() throws Exception {
 
         Assertions.assertTrue(project.resolve("gradlew").toFile().exists(),
                 "Wrapper should exist by default");
+        Assertions.assertTrue(Files.exists(project.resolve("src/main/Docker")),
```

Comment:
The directory name should be `docker` (lower-case), shouldn't it? (This happens several times.. )

Reply:
Yes, you are correct. I'll fix it asap